cmd/osm-webhook/osm-webhook.go

```diff
@@ -0,0 +1,263 @@
+// Command osm-webhook starts up a Kubernetes Validating Webhook on the
```

Comment:
filename and path: s/osm-webhook/osm-validator or something to indicate this is a validating webhook.